docs/howto/catalog_exports.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Data Catalogs Export
+description: This section explains how lakeFS can integrate with external Data Catalogs via metastore update operations. 
+parent: How-To
+---
+
+# Data Catalogs Export
+
+{% include toc_2-3.html %}
+
+## About Data Catalogs Export
+
+Data Catalog Export is all about integrating query engines (like Spark, AWS Athena, Presto, etc.) with lakeFS.
+
+Data Catalogs (such as Hive Metastore or AWS Glue) store metadata for services (such as Spark, Trino and Athena). They contain metadata such as the location of the table, information about columns, partitions and much more.
+
+With Data Catalog Exports, one can leverage the versioning capabilities of lakeFS in external data warehouses and query engines to access tables with branches and commits. 
+
+At the end of this guide, you will be able to query lakeFS data from Athena, Trino and other catalog-dependent tools:
+
+```sql
+use main;
+use my_branch; -- any branch
+use v101; -- or tag
```

Comment:
I prefer to keep SQL keywords in uppercase.  Obviously this is not required -- but ideally we would keep some consistency.  Here I would expect "select * from users" if I said "use main".

```suggestion
USE main;
USE my_branch; -- any branch
USE v101; -- or tag
```